sesame/src/test/scala/org/w3/banana/sesame/io/SesameSerialisationTests.scala

```diff
@@ -1,11 +1,42 @@
 package org.w3.banana.sesame.io
 
+import org.w3.banana.Prefix
 import org.w3.banana.io._
 import org.w3.banana.sesame._
+
 import scala.util.Try
 import org.w3.banana.util.tryInstances._
-
-class SesameTurtleTests extends TurtleTestSuite[Sesame, Try]
+import scalaz.syntax._, comonad._
+
+class SesameTurtleTests extends TurtleTestSuite[Sesame, Try] {
+
+  "write with prefixes" in {
+    val prefix = Prefix[Sesame]("foo", "http://purl.org/dc/elements/1.1/")
+
+    //    val expectedString =
+    //      """
+    //        |@prefix foo:   <http://purl.org/dc/elements/1.1/> .
+    //        |
+    //        |<http://www.w3.org/2001/sw/RDFCore/ntriples/>
+    //        |        foo:creator    "Dave Beckett" , "Art Barstow" ;
+    //        |        foo:publisher  <http://www.w3.org/> .""".stripMargin
+
+    val withPrefix = writer.asString(referenceGraph, "", prefix).copoint
+    withPrefix should include("@prefix foo:")
+    withPrefix should include("foo:creator")
+    withPrefix should include("foo:publisher")
+    withPrefix should not include ("<http://purl.org/dc/elements/1.1/creator>")
+    withPrefix should not include ("<http://purl.org/dc/elements/1.1/publisher>")
+
+    val noPrefix = writer.asString(referenceGraph, "").copoint
```

Comment:
cute the use of `copoint`. I am not against it, but I am not sure it brings much in addition to `.get` which will be more widely known. It's useful in that it brings people's attention to this feature. But I think one would use .copoint in circumstances where one was dealing very generically with Comonads.

IntelliJ tells me that `cobind` is used somewhere in `CommonBindersTest` but it's not easy to find. It would make sense there as that is a very generic library, that is trying to abstract over all types of monads and comonads. 

Does anyone know where methods from the implicit `org.w3.banana.tryinstances.TryInstance` is being used? Or the trick to find out?